### PR TITLE
[WIP] Initial implementation order by fiat value (Do not merge)

### PR DESCRIPTION
**Pull request by vikmeup**

### CODE REVIEW

**Comment by vikmeup** [comment]
@OlegGordiichuk can you take a look?

**Comment by OlegGordiichuk** [comment]
Hmm I will do this after transaction screen refactoring.

File: Trust/Tokens/ViewModels/TokensViewModel.swift [hunk 1]
```diff
@@ -15,7 +15,7 @@ struct TokensViewModel {
 
     var tokensNetwork: TokensNetworkProtocol
 
-    let tokens: Results<TokenObject>
+    var tokens: [TokenObject] = []
```
Comment (OlegGordiichuk) [inline Trust/Tokens/ViewModels/TokensViewModel.swift]: It is a bit wrong way. Hmm we should find more sophisticated way.

**Comment by vikmeup** [comment]
we can leave this for later

**Pull request closed without merging**